src/utils/elementDistanceFromTop.js

```diff
@@ -0,0 +1,11 @@
+export default (element) => {
```

Comment:
Consider using a memo here (though there is probably no real need to optimize this).